pretix_sepadebit/models.py

```diff
@@ -15,3 +16,21 @@ class SepaExportOrder(models.Model):
     order = models.ForeignKey('pretixbase.Order', on_delete=models.CASCADE)
     payment = models.ForeignKey('pretixbase.OrderPayment', on_delete=models.CASCADE, null=True)
     amount = models.DecimalField(max_digits=10, decimal_places=2)
+
+
+class SepaDueDate(models.Model):
+    REMINDER_OUTSTANDING = "o"
+    REMINDER_PROVIDED = "p"
+    REMINDER_CHOICE = (
+        (REMINDER_OUTSTANDING, _("outstanding")),
+        (REMINDER_PROVIDED, _("provided")),
+    )
+
+    payment = models.OneToOneField('pretixbase.OrderPayment', on_delete=models.CASCADE, null=True, related_name='due')
+    date = models.DateField()
+    reminder = models.CharField(
```

Comment:
Hmm, do you see a benefit of doing a choice field here as opposed to just a BooleanField?

Reply:
I wanted to track failures of the reminders as well. And realised to late that it will be of limited use and never rolled it back to a simple Boolean. Will change